cmd/litewitness/litewitness.go

```diff
@@ -102,11 +201,13 @@ func main() {
 			}
 			e <- errors.New("couldn't connect to any bastion")
 		}()
-	} else {
+	} else if !*noListenFlag {
 		go func() {
 			slog.Info("listening", "addr", *listenFlag)
 			e <- srv.ListenAndServe()
 		}()
+	} else if len(logBastions) == 0 {
+		fatal("configured to not open a listening port, but no bastions configured")
```

Comment:
Not sure if this should be a fatal error, or a warning or something configurable.